daemon/top_unix.go

```diff
@@ -5,12 +5,25 @@ package daemon
 import (
 	"fmt"
 	"os/exec"
+	"regexp"
 	"strconv"
 	"strings"
 
 	"github.com/docker/engine-api/types"
 )
 
+func validatePSArgs(psArgs string) error {
+	re := regexp.MustCompile("-o\\s*(.*)=\\s*PID")
+	group := re.FindStringSubmatch(psArgs)
```

Comment:
In fact you can relabel the columns in several ways using UTF8 whitespace eg `docker top 7a709f6ac6ae axe -o 'uid=UID PID' -o pid=COMMAND -o command=`